### ISSUE TITLE
rewardRecipient

### ISSUE THREAD

**Issue by knilecrack** [z-classic/z-nomp#344]
Can you use t_address from CEX as rewardRecipients address? 
I mean I used it and it went completely to different address?

**Comment by przpl** [comment]
No, you can't. All addresses have to be under your control. You need to generate them in daemon.

**Comment by knilecrack** [comment]
Before I tried with CEX t_Address, I've used t address that I generated on completely different daemon and it went through.

**Comment by knilecrack** [comment]
Even if it can't, why did it sent to completely unknown address

**Comment by TheComputerGenie** [comment]
When it comes to your coin config:

"_rewardRecipients_" addresses can be a transparent address of any ownership; this part is nothing more than a standard transaction.

Because of built-in shielding, "_address_", "_zAddress_", and "_tAddress_" must be owned by the daemon in order to send out payments.

**Comment by TheComputerGenie** [comment]
When it comes to your coin config:

"_rewardRecipients_" address(es) can be a transparent address of any ownership; this part is nothing more than a standard transaction.

Because of built-in shielding, "_address_", "_zAddress_", and "_tAddress_" must be owned by the daemon in order to send out payments.

**Comment by knilecrack** [comment]
I understand all that, and for the 1st couple of blocks it worked normally. Then when I changed reward address to go directly to t3 address on CEX it went to some t1 address that is not part of my wallet.

**Comment by TheComputerGenie** [comment]
No CEX address is part of "your wallet", ever (banks just don't work that way).
That being said, you're going to have to be far more specific about what you want to do and what you are experiencing, because I think your confusing yourself and anyone reading what you type.

**Comment by knilecrack** [comment]
I used address from CEX as a rewardRecipient address in pool_configs/zcash.json. and instead of zcash going to that address, when zcash block is mined, it went completely to different address, and that "t" address IS not in any of my wallets. Do you understand me now?

From previous experience, when I first started the mining pool, I used t1 address from different wallet as a rewardRecipient address and it went through.
